dashboard/src/components/AppBar.js

```diff
@@ -9,34 +9,38 @@ import { withStyles } from "@material-ui/core/styles";
 import MenuIcon from "@material-ui/icons/Menu";
 import EnvironmentLabel from "/components/EnvironmentLabel";
 import Wordmark from "/icons/SensuWordmark";
+import Drawer from "/components/Drawer";
 
 class AppBar extends React.Component {
   static propTypes = {
     classes: PropTypes.object.isRequired,
+    viewer: PropTypes.object,
     environment: PropTypes.object,
-    toggleToolbar: PropTypes.func.isRequired,
+    loading: PropTypes.bool.isRequired,
   };
 
-  static defaultProps = { environment: null };
+  static defaultProps = { environment: null, viewer: null };
 
   static fragments = {
+    viewer: gql`
+      fragment AppBar_viewer on Viewer {
+        ...Drawer_viewer
+      }
+      ${Drawer.fragments.viewer}
+    `,
+
     environment: gql`
       fragment AppBar_environment on Environment {
         ...EnvironmentLabel_environment
+        ...Drawer_environment
       }
 
       ${EnvironmentLabel.fragments.environment}
+      ${Drawer.fragments.environment}
     `,
   };
 
   static styles = theme => ({
-    container: {
-      paddingTop: "env(safe-area-inset-top)",
-      backgroundColor: theme.palette.primary.dark,
```

Comment:
This padding was for devices with a notch, I think looks like it's been lost to the re-organization.

Reply:
My mistake. I Thought `env(safe-area-inset-top)` was broken functionality in the style build system. I can revert this.